biz.aQute.bndlib/src/aQute/bnd/service/result/Err.java

```diff
@@ -165,8 +165,18 @@ public V orElseGet(Supplier<? extends V> orElseSupplier) {
 		}
 	}
 
+	@Override
+	public void accept(ConsumerWithException<? super V> ok, ConsumerWithException<? super E> err) {
```

Comment:
Need to add `requireNonNull(ok);` for consistent behavior regardless of whether the Result is Ok or Err.